openrr-apps/README.md

```diff
@@ -244,6 +244,21 @@ Change urdf path and joystick settings (see [here](#joystick)) in [the setting f
 openrr_apps_robot_teleop --config-path=./openrr-apps/config/pr2_teleop_config_ros.toml
 ```
 
+### iRobot Create
+
+- Build again to feature ROS2.
+
+```bash
+cargo build --release --feature ros2
+```
+
+- Run teleop.
+
+```bash
+openrr_apps_robot_teleop \
+  --config-path ./openrr-apps/config/create_teleop_config_ros2.toml
+```
+
```

Comment:
Could you add some description of what will do in the teleop mode? (Undock by ros2 command, so on.)

Reply:
Thank you. I updated these files.